examples/batch/argo-workflows-batch/README.ipynb

```diff
@@ -0,0 +1,459 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">FileNotFoundError: [Errno 2] No such file or directory: 'assets/input-data.txt'
</pre><p> </p><p>Seems <code>assets</code>  folder is not there and no commands create it.</p><p>Could add empty folder creating empty file <code>assets/.keep</code>.</p><p> </p><p>Also I suspect you forgot to commit <code>Makefile</code> that  converts <code>.ipynb</code>  to <code>.md</code>?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/1915/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Good catch! I didn't add a makefile but definitely forgot the mkdir -p assets, will add it!

Reply:
Done